ui/src/core/xibo-cms.js

```diff
@@ -3745,10 +3745,19 @@ function initJsTreeAjax(container, id, isForm, ttl)
 }
 
 function adjustDatatableSize (reload) {
+
+    // Display Map Resize
+    function resizeDisplayMap() {
+        if (typeof refreshDisplayMap === "function") {
+            refreshDisplayMap();
+        }
+    }
+
     reload = (typeof reload == 'undefined') ? true : reload;
     // Shrink table to ease animation
     if($('#grid-folder-filter').is(":hidden")) {
         $('#datatable-container').addClass('col-sm-10').removeClass('col-sm-12');
+        refreshDisplayMap();
```

Comment:
Shouldn't this one be `resizeDisplayMap()` instead?

Reply:
yeah, you are right, my mistake, thanks